tools/coverage/calc/base.go

```diff
@@ -79,6 +59,26 @@ func (blk *codeBlock) addToGroupCov(g *CoverageList) (isConcerned bool) {
 	return true
 }
 
+// add blk Coverage to file group Coverage; return true if the row is concerned
```

Comment:
Funny, this is exactly the same comment used in `addToGroupCov`. @steuhs what's the right comment for this function?